tutorials/plot_object_annotations.py

```diff
@@ -0,0 +1,170 @@
+        experimental condition events, etc. MNE represents events with integers
+        that are stored in numpy arrays of shape (n_events, 3). Such arrays are
+        classically obtained from a trigger channel, also referred to as stim
+        channel.
+
+    annotations
+        An annotation is defined by an onset, a duration, and a string
+        description. It can contain information about the experiments, but
+        also details on signals marked by a human: bad data segments,
+        sleep scores, sleep events (spindles, K-complex) etc.
+
+Both events and :class:`Annotations <mne.Annotations>` can be seen as triplets
+where the first element answers to **when** something happens and the last
+element refers to **what** it is.
+The main difference is that events represent the onset in samples taking into
+account the first sample value
+(:attr:`raw.first_samp <mne.io.Raw.first_samp>`), and the description is
+an integer value.
+In contrast, :class:`Annotations <mne.Annotations>` represents the
+``onset`` in seconds (relative to the reference ``orig_time``),
+and the ``description`` is an arbitrary string.
+There is no correspondence between the second element of events and
+:class:`Annotations <mne.Annotations>`.
+For events, the second element corresponds to the previous value on the
+stimulus channel from which events are extracted. In practice, the second
+element is therefore in most cases zero.
+The second element of :class:`Annotations <mne.Annotations>` is a float
+indicating its duration in seconds.
+
+See :ref:`sphx_glr_auto_examples_io_plot_read_events.py`
+for a complete example of how to read, select, and visualize **events**;
+and :ref:`sphx_glr_auto_tutorials_plot_artifacts_correction_rejection.py` to
+learn how :class:`Annotations <mne.Annotations>` are used to mark bad segments
+of data.
+
+An example of events and annotations
+------------------------------------
+
+The following example shows the recorded events in `sample_audvis_raw.fif` and
+marks bad segments due to eye blinks.
+"""
+
+import os.path as op
+import numpy as np
+
+import mne
+
+# Load the data
+data_path = mne.datasets.sample.data_path()
+fname = op.join(data_path, 'MEG', 'sample', 'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(fname)
+
+# Plot the events
+events = mne.find_events(raw)
+
+# Specify event_id dictionary based on the experiment
+event_id = {'Auditory/Left': 1, 'Auditory/Right': 2,
+            'Visual/Left': 3, 'Visual/Right': 4,
+            'smiley': 5, 'button': 32}
+color = {1: 'green', 2: 'yellow', 3: 'red', 4: 'c', 5: 'black', 32: 'blue'}
+
+mne.viz.plot_events(events, raw.info['sfreq'], raw.first_samp, color=color,
+                    event_id=event_id)
+
+# Create some annotations specifying onset, duration and description
+annotated_blink_raw = raw.copy()
+eog_events = mne.preprocessing.find_eog_events(raw)
+n_blinks = len(eog_events)
+# Center to cover the whole blink with full duration of 0.5s:
```

Comment:
this may be a bit misleading - the events are centered and we are actually de-cenetering them by pushing them back in time to mark onset, not event center.